### ISSUE TITLE
Color setting

### ISSUE THREAD

**Issue by d-albrecht** [Sesu8642/FeudalTactics#24]
Would it be possible to have a global game settings menu where I can choose my color out of the active color scheme? I'm not a fan of always playing blue.

Furthermore, have you made sure that the color scheme is appropriate when it comes to users with any colorblindness / visual impairment? I don't suffer any of these issues, but I know that it is really easy to fall into this trap and choose colors that some might not be able to distinguish (and being called out for that). In case you're unsure, use the above settings menu to offer a range of color schemes to choose from so everyone will find some working scheme.

**Comment by Sesu8642** [comment]
Yes, I did think of that but didn't implement it (yet) because it's quite a lot of work. Not the highest priority right now.

**Comment by bodqhrohro** [comment]
My nephew asks how to play for pink rather than for blue every time they see the game. Pretty sad it's not supported.

**Comment by gwhitney** [comment]
Most importantly I would very much like the ability to replay a given starting position in control of a different color in that starting position. When I play a very close/interesting game I often wonder if I could have won it "from the other guy's position". Thanks for considering.

**Comment by Sesu8642** [comment]
@gwhitney interesting idea, I like it. I'm unsure if it should be linked to the color setting though.

One idea to implement would be to give the option to tap/click on any tile of the generated map before starting the game. Then some indicator for the selected color would be needed.

**Comment by gwhitney** [comment]
Oh I guess I misunderstood,I thought this issue was precisely the choice of which team/color to play.

**Comment by Sesu8642** [comment]
I believe this issue is about a global setting for choosing a color to always play as. The game would then just color the map differently but the starting situation of a given map would not be affected. At least this is how I imagined the implementation.

**Comment by gwhitney** [comment]
Ah ok. Well, either you could kill two birds with one stone and keep the colors the same and just allow the player to choose which one to play, or this could be just about what color to assign to each position, and you or I could open a separate issue about choosing which position to play in a given starting configuration. Just let me know.

**Comment by Sesu8642** [comment]
I'll decide when I work on this and potentially create a new issue then. Thanks for your idea :)

**Comment by AnthonyClayton12** [comment]
@Sesu8642 I have created a new button on the game setup screen to allow the user to choose the color of their kingdom. I will send you more information and a pull request.

**Comment by Sesu8642** [comment]
@AnthonyClayton12 opened #73. The setting only affects the color, not the gameplay.
I'm unsure if this is something people would want to change with the map parameters. Maybe it should be in the main preferences instead. Looking for feedback from the other participants here :)

**Comment by gwhitney** [comment]
I agree it's something that I'd be unlikely to change on a map-by-map basis; it's more like I would set my favorite color once and for all. So by that rationale, I'd agree that it should be on the preferences screen that you get to from the main menu.

The other feature, which was my alternate interpretation of this item, where you pick which one of the six initial territories you'd like to play with, would of course need to be on the map parameters screen.
